components/monitoring/grafana/base/grafana-app.yaml

```diff
@@ -135,7 +135,7 @@ spec:
         - '-http-address='
         - '-email-domain=*'
         - '-upstream=http://localhost:3000'
-        - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+        - '-openshift-sar={"resource": "namespaces", "resourceName": "appstudio-grafana", "namespace": "appstudio-grafana", "verb": "get"}'
```

Comment:
I think that we should also add RBAC permissions so RHTAP developers can view Grafana.
You can add it in the `components/authentication/base/everyone-can-view.yaml` file.